### PR TITLE
Features/cd

### PR DESCRIPTION
**Pull request by CMoucer**
Add randomized coordinate descent + documentation.

### CODE REVIEW

**Review by AdrienTaylor — COMMENTED** [review]
Hello!
minor comments:
- update tests?
- questions: second example in folder "potential functions"? (maybe not)
- rename folder: stochastic -> stochastic and randomized?

I'll check the rest tomorrow :)

File: PEPit/examples/stochastic_convex_minimization/__init__.py [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
+from .randomized_coordinate_descent_smooth_strongly_convex import wc_randomized_coordinate_descent_smooth_strongly_convex
+from .randomized_coordinate_descent_smooth_convex import wc_randomized_coordinate_descent_smooth_convex
+
 
 __all__ = ['point_saga', 'wc_point_saga',
            'saga', 'wc_saga',
            'sgd', 'wc_sgd',
            'sgd_overparametrized', 'wc_sgd_overparametrized',
-           ]
+          'randomized_coordinate_descent_smooth_strongly_convex', 'randomized_coordinate_descent_smooth_convex'
```
Comment (bgoujaud) [inline PEPit/examples/stochastic_convex_minimization/__init__.py:13]: Please keep the same convention as above by adding functions as well

File: docs/source/examples/d.rst [hunk 2]
```diff
@@ -24,3 +24,12 @@ SAGA
 Point SAGA
 ----------
 .. autofunction:: PEPit.examples.stochastic_convex_minimization.wc_point_saga
+
```
Comment (bgoujaud) [inline docs/source/examples/d.rst:27]: Please try to leave 2 blank lines in between examples

**Review by bgoujaud — COMMENTED** [review]
I made very minor changes, nothing important.
The rest looks good. Tests run and doc compiles as expected.

**Review by AdrienTaylor — APPROVED** [review]
Please carefully recompile & check documentation.

It was very inconsistent (changes of index: t, k, n and bad algorithms names and definitions).
Normally, I've corrected it, but proofread before merge.

**Pull request merged**